aliyun_driver.go

```diff
@@ -809,6 +811,14 @@ func (d *Driver) Remove(ctx context.Context, info *types.ClusterInfo) error {
 	if err != nil {
 		return err
 	}
+
+	// Validate if the cluster info has clusterID
+	err = ReflectStructField(state, "ClusterID")
```

Comment:
I don't think this function is necessary. We can check it by
```
if state ==nil || state.ClusterID ==""
```

Reply:
updated.